entity-service/src/main/resources/configs/common/application.conf

```diff
@@ -7,6 +7,7 @@ entity.service.config = {
     dataStoreType = mongo
     mongo {
       host = localhost
+      host = ${?MONGO_HOST}
       port = 27017
```

Comment:
just wondering, should we have env variable for mongo port as well?

Reply:
Had the same thought, but till now, we didn't have to override it, so, should we?

Reply:
It's ok for now. We can add it later if required.